apps/trading/e2e/README.md

```diff
@@ -0,0 +1,118 @@
+# Trading End-To-End Tests
+
+This direcotry contains end-to-end tests for the trading application using vega-market-sim. This README will guide you through setting up your environment and running the tests.
+
+## Prerequisites
+
+- [Poetry](https://python-poetry.org/docs/#installing-with-the-official-installer)
+- [Docker](https://www.docker.com/)
+- [Python versions ">=3.9,<3.11"](https://www.python.org/)
+
+## Getting Started
+
+1. **Install Poetry**: Follow the instructions on the [official Poetry website](https://python-poetry.org/docs/#installing-with-the-official-installer).
+2. **Install Docker**: Follow the instructions on the [official Docker website](https://docs.docker.com/desktop/).
+3. **Install Python**: Follow the instructions on the [official Python website](https://www.python.org/)
+   **ensure you install a version between 3.9 and 3.11.**
+4. **Start up a Poetry environment**: Execute the commands below to configure the Poetry environment.
+
+```bash
+poetry shell
+```
+
+5. **Install python dependencies**
+
+```bash
+poetry install
+```
+
+6. **Install Playwright Browsers**: Execute the command below to browsers for Playwright.
+
+```bash
+playwright install chromium
+```
+
+7. **Download necessary binaries**:
+   Use the following command within your Python environment. The `--force` flag ensures the binaries are overwritten, and the `--version` specifies the desired version. e.g. `v0.73.1`
+
+```bash
+python -m vega_sim.tools.load_binaries --force --version $VEGA_VERSION
+```
+
+8. **Pull the desired Docker image**
+
+```bash
+docker pull vegaprotocol/trading:develop
+```
+
+9. **Run tests**: Poetry/Python will serve the app from docker
+
+```bash
+poetry run pytest
+```
+
+### Docker images
+
+Pull the desired image:
+
+**Testnet**
+
+```bash
+docker pull vegaprotocol/trading:develop
+```
+
+**Mainnet**
+
+```bash
+docker pull vegaprotocol/trading:main
+```
+
+Find all available images on [Docker Hub](https://hub.docker.com/r/vegaprotocol/trading/tags).
+
+#### Create a Docker Image of Your Locally Built Trading App
+
+To build your Docker image, use the following commands:
+
+```bash
+yarn nx build trading ./docker/prepare-dist.sh docker build -f docker/node-outside-docker.Dockerfile --build-arg APP=trading --build-arg ENV_NAME=stagnet1 -t vegaprotocol/trading:latest .
+```
+
+## Running Tests 🧪
+
+Before running make sure the docker daemon is runnign so that the app can be served.
+
+To run a specific test, use the `-k` option followed by the name of the test.
+
+Run all tests:
+
+```bash
+poetry run pytest
```

Comment:
can we add the instructions to set the correct interpreter given there is a specific poetry one. My VSCode sets to my default installation.